modules/admin-home/components/conversion-banner.php

```diff
@@ -23,7 +24,7 @@ private function is_conversion_banner_active(): bool {
 			return false;
 		}
 
-		if ( Utils::is_elementor_active() ) {
+		if ( Utils::has_pro() ) {
```

Comment:
I think we should keep both checks as I recently learned that some people deactivate Core while keeping Pro active (not sure why, but they do)

Reply:
@nuritsha added